src/common/bit_vector.hpp

```diff
@@ -223,7 +223,8 @@ class BitVector
 
   static void generate_test_instances(std::list<BitVector *> &o);
 private:
-  struct NoInitAllocator : public std::allocator<__u32> {
+  struct NoInitAllocator : public std::allocator<__u32> {A
```

Comment:
might want to remove the trailing "A".

Reply:
Errrgh, where did that come from.
Will update.